packages/jumpstarter-driver-http/jumpstarter_driver_http/driver.py

```
@@ -3,6 +3,7 @@
 from typing import Optional
 
 import anyio
+import anyio.from_thread
```

Pylint (3.3.7)

[error] Unable to import 'anyio.from_thread'

(E0401)